scripts/datamodel.py

```diff
@@ -237,10 +253,18 @@ def load_datamodel_dump(
     for type_, resources in cognite_resources_by_type.items():
         print(f"  {type_}: {len(resources)}")
 
-    space_list = list({r.space for _, resources in cognite_resources_by_type.items() for r in resources})
+    space_list = list(
+        {
+            r.space
+            for _, resources in cognite_resources_by_type.items()
+            for r in resources
+        }
```

Comment:
```suggestion
        {
            r.space
            for resources in cognite_resources_by_type.values()
            for r in resources
        }
```

Reply:
Sorry, auto-merge was on, I will turn that off, it's not particularly good practice unless the change is tiny.